crates/nu-cli/tests/completions/mod.rs

```diff
@@ -773,6 +773,50 @@ fn external_completer_invalid() {
     assert!(suggestions.is_empty());
 }
 
+#[test]
+fn command_wide_completion_external() {
+    let mut completer = custom_completer();
+
+    let sample = /* lang=nu */ r#"
+        @complete external
+        extern "gh" []
+
+        gh alias one two"#;
+
+    let suggestions = completer.complete(sample, sample.len());
+    let got = suggestions
+        .iter()
+        .map(|s| s.value.as_str())
+        .collect::<Vec<_>>();
+    let expacted = vec!["gh", "alias", "one", "two"];
```

Comment:
You can simply this with the helper function of `match_suggestions`.

Also there's a typo in `expected`.